package.json

```diff
@@ -81,9 +81,9 @@
     "@storybook/react": "6.0.28",
     "@testing-library/react": "10.4.9",
     "@testing-library/react-hooks": "5.0.3",
-    "@testing-library/jest-dom": "5.11.9",
+    "@testing-library/jest-dom": "^5.11.10",
     "@types/copy-webpack-plugin": "6.4.0",
-    "@types/jest": "24.9.1",
+    "@types/jest": "^26.0.22",
```

Comment:
I'm surprised this works without TS4... it doesn't use `import/export type` internally ?

Reply:
It was actuall syncpack that upgraded that 😅
Not sure it's gonna work though....

Reply:
Yeah, just confirming @ling1726 , this broke the build 😂😂😂
I'll reverse this